### PR TITLE
Update page in Connect API Manager “subscription-for-the-consumer”

### PR DESCRIPTION
**Pull request by cbordier**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by cbordier** [comment]
@dfeldick @jcollins-axway @kflan-io @vivekschauhan please review

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md [hunk 1]
```diff
@@ -1,139 +1,220 @@
...
+
+Each API can define its own approval mode:
+
+* manual (default): an API provider has to approve the subscription before the consumer receives the API credentials.
+(Optional) the agent configuration contains a webhook information that will be triggered on each subscription state change. The webhook implementation can, for instance, trigger a MS Teams card to a dedicated Teams channel where the API provider will approve the subscription.
+* automatic: the subscription is auto-approve without human intervention
+
+## Supported use cases for receiving API credentials
+
+Once the subscription is approved, the agent catch this event from AMPLIFY Central and, based on its configuration, can either forward the credentials using an SMTP server or using a webhook.
```
Comment (dfeldick) [inline content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md:28]: agent catches

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md [hunk 2]
```diff
@@ -1,139 +1,220 @@
...
+
+* email: the agent configuration contain the access details to an SMTP server (endpoint / prot / credentials if any) and the templates for the emails. Emails can be trigger when subscription succeed, subscription fails or when unsubscribing to an API.  More information about this configuration can be found [here](/docs/central/connect-api-manager/gateway-administation#customizing-smtp-notifications)
```
Comment (dfeldick) [inline content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md:30]: configuration contains

Comment (dfeldick) [inline content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md:30]: emails can be triggered

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md [hunk 3]
```diff
@@ -1,139 +1,220 @@
...
+* webhook: the agent configuration contain the webhook details to send the payload to (catalog asset url / catalog asset name / subscriber email / credentials / action=APPROVE / authtemplate=preconfigure security tenplate sentence)
```
Comment (dfeldick) [inline content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md:31]: configuration contains the webhook details about where to send the payload to

Comment (dfeldick) [inline content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md:31]: security template sentence

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md [hunk 4]
```diff
@@ -1,139 +1,220 @@
...
+                 },
+             },
+             api: {
+                 // custom properties...
+             }
+         }
+     ```
+2. (Optional) An API provider adds API access on the application(s) for each API they wish to allow their subscriber to subscribe to.
+
+3. (Optional) An API provider allow the consumer to create an application during the subscription flow (add the property `APIMANAGER_ALLOWAPPLICATIONAUTOCREATION=true` to the discovery agent configuration file)
```
Comment (dfeldick) [inline content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md:115]: provider allows

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md [hunk 5]
```diff
@@ -1,139 +1,220 @@
...
+   3. Select the Team and API Manager Application name (created in Step 1) for which you want to subscribe. **WARNING**: The subscription will fail if you select an application for which no APIs have been given access. For additional information, see [Manage AMPLIFY Catalog subscriptions.](https://docs.axway.com/bundle/axway-open-docs/page/docs/catalog/manage_subscriptions/index.html)
+
+2. Based on the API subscription approval (manual VS automatic), an API provider has to approve the subscription.
+
+3. The Discovery Agent receives the subscription event:
+
+   * subscription status: **Active**
+
+      * Subscription ID is automatically added to the **Custom** field of the application.
+      * agent trigger credentials sending (either via email or via webhook).
```
Comment (dfeldick) [inline content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md:132]: agent triggers

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md [hunk 6]
```diff
@@ -1,139 +1,220 @@
...
+      * If failure during the process for any reason, set the subscription status to: **Subscription failed**. Refer to the Discovery Agent log for more information. You can delete the subscription and start again from Step 1.
```
Comment (dfeldick) [inline content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md:133]: If a failure occurs during

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md (hunk 2 above)
Comment (cbordier) [reply content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md]: done in last commit

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md (hunk 3 above)
Comment (cbordier) [reply content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md]: done in last commit

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md (hunk 4 above)
Comment (cbordier) [reply content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md]: done in last comment

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md (hunk 6 above)
Comment (cbordier) [reply content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md]: done in last comment

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md (hunk 5 above)
Comment (cbordier) [reply content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md]: done in last commit

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md (hunk 2 above)
Comment (cbordier) [reply content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md]: done in last commit

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md (hunk 3 above)
Comment (cbordier) [reply content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md]: done in last commit

**Review by kflan-io — APPROVED** [review]

**Comment by cbordier** [comment]
@lbadenhop . you can review now.

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**